provider/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptTarget.kt

```diff
@@ -76,6 +80,16 @@ fun projectScriptTarget(project: Project, topLevelScript: Boolean): KotlinScript
         })
 
 
+private
+fun gradleScriptTarget(gradle: Gradle) =
+    KotlinScriptTarget(
+        gradle,
+        type = Gradle::class,
```

Comment:
:o: Is this `Gradle` type used for anything other than Init scripts? If it is then this method might make sense to be named `gradleInitScriptTarget`. If not, this is probably fine.